index.html

```diff
@@ -0,0 +1,442 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Sebastian M - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Sebastian M</h1>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+      <nav class="flex-nav">
+        <a href="#" class="menu-open">☰</a>
+        <div class="menu">
+          <a href="#hello">Hello</a>
+          <a href="#portfolio">Portfolio</a>
+          <a href="#about">About</a>
+          <a href="#contact">Contact</a>
+          <div class="menu-close"><img src="assets/img/Union.svg" alt=""></div>
```

Comment:
- [ ] [OPTIONAL] Consider making the mobile menu fixed with 100% height and width, that way it will:
 - Prevent the user to scroll to a section while the menu is still open.
 - Avoid the menu section to push the content of the page down when it is opened.